native/dgetf2.go

```diff
@@ -0,0 +1,63 @@
+package native
+
+import (
+	"math"
+
+	"github.com/gonum/blas/blas64"
+)
+
+// Dgetf2 computes the LU decomposition of the m×n matrix a.
+// The LU decomposition is a factorization of a into
+//  A = P * L * U
+// where P is a permutation matrix, L is a unit lower triangular matrix, and
+// U is a (usually) non-unit upper triangular matrix. On exit, L and U are stored
+// in place into a.
+//
+// ipiv is a permutation vector. It indicates that row i of the matrix was
+// changed with ipiv[i]. ipiv must have length at least min(m,n), and will panic
+// otherwise.
+//
+// Dgetf2 returns whether the matrix a is singular. The LU decomposition will
```

Comment:
add the name ok to the return - at the moment the documentation make it look like you are returning (isSingular bool).